vowpalwabbit/parse_args.cc

```diff
@@ -1272,82 +1274,134 @@ VW::LEARNER::base_learner* setup_base(options_i& options, vw& all)
     return base;
 }
 
+void create_reduction_template(options_i& options, vw& all)
+{
+  static const std::unordered_set< VW::LEARNER::base_learner* (*)(VW::config::options_i&, vw&)> multi = {
+    explore_eval_setup
+    , cbifyldf_setup
+    , cbify_setup
+    , warm_cb_setup
+    , VW::slates::slates_setup
+    , CCB::ccb_explore_adf_setup
+    , VW::shared_feature_merger::shared_feature_merger_setup
+    , cb_sample_setup
+    , cb_dro_setup
+    , VW::cb_explore_adf::greedy::setup
+    , VW::cb_explore_adf::softmax::setup
+    , VW::cb_explore_adf::rnd::setup
+    , VW::cb_explore_adf::regcb::setup
+    , VW::cb_explore_adf::first::setup
+    , VW::cb_explore_adf::cover::setup
+    , VW::cb_explore_adf::bag::setup
+    , cb_adf_setup
+    , cb_adf_setup
```

Comment:
Why are only a subset of the reductions here?

Reply:
might be related to the function apply_from

Reply:
These are the reductions that require a multilearner, its used to determine which base reduction needs to be used (single vs multi)